examples/podman/stepca.run.md

```diff
@@ -0,0 +1,45 @@
+## Example creation of a Podman container & secret
+
+* Using a [cryptographically strong secret from `openssl` with an `8192` character `hex` string](https://docs.podman.io/en/latest/markdown/podman-secret-create.1.html#examples)
+
+see also:
+
+- [Create a "quadlet"](https://github.com/containers/podlet)
+- [Netbird VPN](https://github.com/netbirdio/netbird)
```

Comment:
Why is this here? There's only a comment using `netbird` below, but it doesn't seem to be required for this example.

Reply:
My comment was specifically for Netbird. I understand that running the CA on a VPN interface is a nice way to add additional security, but IMO the link to `netbird` isn't required.

Reply:
ok will remove the link after work & resolve

Reply:
added a `mkdir -p` to the `run` example for persistent storage

Reply:
& a link back to your [TPM Blog](https://smallstep.com/blog/trusted-platform-modules-tpms/) in a note about `--privileged`

Reply:
& `ReloadSignal=SIGHUP` to the quadlet

![reload](https://github.com/user-attachments/assets/1704e3d0-27cc-48d6-b52f-ecc2ab444f26)

Reply:
* also removed the `finger.print` file that got accidentally added to the PR